src/App.vue

```diff
@@ -96,18 +96,25 @@ export default {
 
 		// Load manifest
 		httpClient.get(this.$root.manifestUrl).then((response) => {
-			this.$root.manifest = response.data;
+			const manifest = response.data;
+			const manifestValid = this.$root.validateManifest(manifest);
 
-			// Merging user-set query params with defaults
-			this.$root.params = this.$root.getParams();
+			if (manifestValid) {
+				this.$root.manifest = manifest;
 
-			window.addEventListener('popstate', () => {
+				// Merging user-set query params with defaults
 				this.$root.params = this.$root.getParams();
-			});
 
-			if (this.$root.options.title) {
-				window.document.title = `${this.$root.convertValueToArray(this.$root.manifest.label)[0]}`
-					+ ` | ${this.$root.options.title}`;
+				window.addEventListener('popstate', () => {
+					this.$root.params = this.$root.getParams();
+				});
+
+				if (this.$root.options.title) {
+					window.document.title = `${this.$root.convertValueToArray(this.$root.manifest.label)[0]}`
+							+ ` | ${this.$root.options.title}`;
+				}
+			} else {
+				this.$root.error = 'Error validating IIIF manifest: TIFY only supports IIIF Presentation API 2.0';
```

Comment:
Maybe a more general error message? We don't know whether it's an older or newer manifest or even some completely different file.

Reply:
I also had this in mind but then decided to describe what API is expected because it's not clear if a general error would help here. What would be your suggestion?

Reply:
Something like "Please provide a valid IIIF Presentation API 2.x Manifest"